### PR TITLE
feat(style): remove ovh-form-flat from line phone section.

### PR DESCRIPTION
**Pull request by frenautvh**

### CODE REVIEW

File: client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html [hunk 1]
```diff
@@ -332,24 +336,20 @@ <h3 data-translate="telephony_line_phone_order_summary"></h3>
                          data-contracts-validated="PhoneOrderCtrl.order.isContractsAccepted">
                     </div>
 
-                    <div data-ng-if="!PhoneOrderCtrl.phone">
+                    <div data-ng-if="true || !PhoneOrderCtrl.phone">
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/order/telecom-telephony-line-phone-order.html]: -`true`

File: client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html [hunk 2]
```diff
@@ -20,131 +19,125 @@
         </h3>
         <!-- SURNAME -->
         <div class="form-group">
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html]:
```html
data-ng-class="{
    'has-error': addPhonecontactForm.surname.$touched && addPhonecontactForm.surname.$invalid
}"
```

File: client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html [hunk 3]
```diff
@@ -20,131 +19,125 @@
...
+                name="surname"
+                id="surname"
+                class="form-control"
+                autofocus
+                required
+                data-ng-model="ContactAddCtrl.phonecontactForm.surname"
+                data-ng-pattern="/^[\w\s.\-]*$/"
+                data-ng-maxlength="246"
+                data-translate-attr="{ placeholder: 'telephony_phonebook_contact_add_surname_placeholder' }" />
+            <div role="alert"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html]:
```html
<span class="help-block"
      role="alert"
      data-ng-if="addPhonecontactForm.surname.$touched"
      data-ng-messages="addPhonecontactForm.surname.$error">
    <span data-ng-message="required"
       data-translate="telephony_phonebook_contact_add_surname_error_required">
    </span>
    <span data-ng-message="pattern"
          data-translate="telephony_phonebook_contact_add_surname_error_pattern">
    </span>
    <span data-ng-message="maxlength"
          data-translate="telephony_phonebook_contact_add_surname_error_maxlength">
    </span>
</span>
```

File: client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html [hunk 4]
```diff
@@ -20,131 +19,125 @@
...
-                    <p class="text-danger"
-                       data-ng-message="pattern"
-                       data-translate="telephony_phonebook_contact_add_name_error_pattern">
-                    </p>
-                    <p class="text-danger"
-                       data-ng-message="maxlength"
-                       data-translate="telephony_phonebook_contact_add_name_error_maxlength">
-                    </p>
-                </div>
+            <label for="name"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html]: Same behavior as `surname` field.

File: client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html [hunk 5]
```diff
@@ -20,131 +19,125 @@
...
-                     data-ng-messages="addPhonecontactForm.group.$error">
-                    <p class="text-danger"
-                        data-ng-message="pattern"
-                        data-translate="telephony_phonebook_contact_add_group_error_pattern">
-                    </p>
-                    <p class="text-danger"
-                       data-ng-message="maxlength"
-                       data-translate="telephony_phonebook_contact_add_group_error_maxlength">
-                    </p>
+            <label for="group"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html]: Same behavior as `surname` field.

File: client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html [hunk 6]
```diff
@@ -157,56 +150,52 @@ <h5 class="mt-3 text-info">
...
-                            data-translate-attr="{ placeholder: 'telephony_phonebook_contact_column_home_phone_title' }" />
-                        <div role="alert"
-                             data-ng-if="addPhonecontactForm.homePhone.$dirty"
-                             data-ng-messages="addPhonecontactForm.homePhone.$error">
-                            <p class="text-danger"
-                               data-ng-message="phoneNumber"
-                               data-translate="telephony_phonebook_contact_add_error_phonenumber">
-                            </p>
-                        </div>
+                    <label for="homePhone"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html]: Same behavior as `surname` field.

File: client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html [hunk 7]
```diff
@@ -157,56 +150,52 @@ <h5 class="mt-3 text-info">
...
-                            data-translate-attr="{ placeholder: 'telephony_phonebook_contact_column_home_mobile_title' }" />
-                        <div role="alert"
-                             data-ng-if="addPhonecontactForm.homeMobile.$dirty"
-                             data-ng-messages="addPhonecontactForm.homeMobile.$error">
-                            <p class="text-danger"
-                               data-ng-message="phoneNumber"
-                               data-translate="telephony_phonebook_contact_add_error_phonenumber">
-                            </p>
-                        </div>
+                    <label for="homeMobile"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html]: Same behavior as `surname` field.

File: client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html [hunk 8]
```diff
@@ -216,56 +205,52 @@ <h5 class="mt-3 text-info">
...
-                            data-translate-attr="{ placeholder: 'telephony_phonebook_contact_column_work_phone_title' }" />
-                        <div role="alert"
-                             data-ng-if="addPhonecontactForm.workPhone.$dirty"
-                             data-ng-messages="addPhonecontactForm.workPhone.$error">
-                            <p class="text-danger"
-                               data-ng-message="phoneNumber"
-                               data-translate="telephony_phonebook_contact_add_error_phonenumber">
-                            </p>
-                        </div>
+                    <label for="workPhone"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html]: Same behavior as `surname` field.

File: client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html [hunk 9]
```diff
@@ -216,56 +205,52 @@ <h5 class="mt-3 text-info">
...
-                            data-translate-attr="{ placeholder: 'telephony_phonebook_contact_column_work_mobile_title' }" />
-                        <div role="alert"
-                             data-ng-if="addPhonecontactForm.workMobile.$dirty"
-                             data-ng-messages="addPhonecontactForm.workMobile.$error">
-                            <p class="text-danger"
-                               data-ng-message="phoneNumber"
-                               data-translate="telephony_phonebook_contact_add_error_phonenumber">
-                            </p>
-                        </div>
+                    <label for="workMobile"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/phone/phonebook/contact-add/telecom-telephony-line-phone-phonebook-contact-add.html]: Same behavior as `surname` field.

File: client/app/telecom/telephony/line/phone/mgcpIpRestriction/telecom-telephony-line-phone-mgcpIpRestriction.html [hunk 10]
```diff
@@ -108,10 +109,11 @@ <h2 data-translate="loading"></h2>
                                 data-ng-disabled="mgcpDefaultIpRestrictionForm.$invalid ||
                                                   !MgcpIpRestrictionCtrl.hasMgcpDefaultIpRestrictionChanged() ||
                                                   MgcpIpRestrictionCtrl.isChangingMgcpDefaultIpRestriction">
+                                <bouncing-box-loader class="d-inline-block align-middle mr-2"
```
Comment (feoche) [inline client/app/telecom/telephony/line/phone/mgcpIpRestriction/telecom-telephony-line-phone-mgcpIpRestriction.html]: Are these classes necessary here with the new `bouncing-box-loader` tag?

Comment (antleblanc) [reply client/app/telecom/telephony/line/phone/mgcpIpRestriction/telecom-telephony-line-phone-mgcpIpRestriction.html]: Yes they are! By default it's computed as a block element.

**Pull request merged**